var/spack/repos/builtin/packages/suite-sparse/package.py

```diff
@@ -75,47 +83,28 @@ def install(self, spec, prefix):
             # completely disabled. See
             # [SuiteSparse/SuiteSparse_config/SuiteSparse_config.mk] for more.
             'CUDA=no',
-            'CUDA_PATH={0}'.format(
```

Comment:
Do you have a reason to prefer `format`? Using format will spread the logic over 3 lines, 2 if I introduce a separate `cuda_path` variable.